Assets/TestCubes/Scripts/UI/Controllers/ImagesWindowController.cs

```diff
@@ -75,7 +78,8 @@ private async UniTask DownloadItem(ImageItem item)
             item.SetTexture(((DownloadHandlerTexture)request.downloadHandler).texture);
 
             ImageItemView itemView = _itemViews[item.Id];
-            if (itemView != null)
+            // Note: Just reminder that we can cast Object to bool to check for aliveness (JS like)
```

Comment:
Actually, we shouldn't, because == is a correct nullcheck for GameObjects (because only == is overriden and checking object state on backing scripting side https://stackoverflow.com/questions/72069139/unity-this-null-returns-true-how-can-this-happen )

Reply:
yeah == is a correct nullcheck. I just say there is another way to check. Which to use, you can decide